Issue 10: use s3mock

ravibeta opened the pull request:
**Change log description**  
fixes checkstyle

**Purpose of the change**  
Fixes #10 

**What the code does**  
allows import

**How to verify it**  
gradlew checkstyletest
https://travis-ci.com/github/ravibeta/pravega-storage-adapters-1/builds/182942170

Commit:
add checkstyle fix

derekm commented:
This may related to #10, but it doesn't really fix #10... Merging this as-is would accidentally close #10?

ravibeta commented:
Yes Derek

Commit:
switch to s3Mock

Commit:
fix travis
export GRADLE_OPTS="-Xms128m"
$ javac -J-Xmx32m -version
javac 1.8.0_252
0.01s$ ./gradlew assemble
Error occurred during initialization of VM
Initial heap size set to a larger value than the maximum heap size
The command "./gradlew assemble" failed and exited with 1 during .
Your build has been stopped.

Commit:
fix checkstyle due to new packages

ravibeta committed:
'1'

ravibeta committed:
add checkstyletest

ravibeta committed:
add methods

ravibeta committed:
add methods

Signed-off-by: ravibeta <ravibeta@hotmail.com>
